backend/image/forms.py

```diff
@@ -31,14 +31,57 @@ def value_from_datadict(self, data, files, name):
 
 
 class ImageAdminForm(forms.ModelForm):
-    file = SVGAndImageFormField(widget=SVGAndImageInput)
+    file = SVGAndImageFormField(
+        widget=SVGAndImageInput, help_text="Upload an image file (supported formats: JPG, JPEG, PNG, GIF, SVG, WEBP)"
+    )
+
+    title = forms.CharField(
+        max_length=255, help_text="A descriptive title for the image that will be used for administrative purposes"
+    )
+
+    description = forms.CharField(
+        required=False,
+        widget=forms.Textarea,
+        help_text="A detailed description of the image content - useful for content management",
+    )
+
+    alt = forms.CharField(
+        max_length=255,
+        required=False,
+        help_text="Alternative text that describes the image - important for accessibility and SEO. "
```

Comment:
```suggestion
        help_text="Alternative text that describes the image. "
```
IMO, this part is too technical. With the next sentence, it's already crystal clear.

Reply:
```suggestion
        help_text="Alternative text that describes the image - important for accessibility and SEO. This text is read aloud by screen readers or whenever the image cannot be loaded."
```